x/tokenomics/types/params.go

```diff
@@ -4,6 +4,15 @@ import (
 	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
 )
 
+// DaoRewardAddress is the address that will receive the dao/foundation rewards
+// during claim settlement (global mint TLM). It is intended to be assigned via
+// -ldflags="-X github.com/pokt-network/poktroll/app.FoundationRewardAddress=<foundation_address>"
```

Comment:
In `config.yaml`, the name is `DaoRewardAddress` and here its `FoundationRewardAddress`. 

Per my other comment, I would like to avoid merging this flag-based approach to main altogether.